src/test/unit/common/configuration.test.ts

```diff
@@ -77,6 +77,13 @@ suite('Configuration', () => {
     strictEqual(configuration.snykCodeBaseURL, 'http://deeproxy.custom.endpoint.com');
   });
 
+  test('Snyk Code: base url respects single tenant endpoint configuration', () => {
+    const workspace = stubWorkspaceConfiguration(ADVANCED_CUSTOM_ENDPOINT, 'https://app.custom.snyk.io/api');
+    const configuration = new Configuration({}, workspace);
+
+    strictEqual(configuration.snykCodeBaseURL, 'https://deeproxy.custom.snyk.io');
+  });
+
```

Comment:
Nice, let's have another test for `Configuration.snykCodeUrl` property :)

Reply:
sounds good, done with f28b74abe9965a3574c23f10497afc61fc201195